src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -43,6 +44,7 @@ interface CheckoutState {
 }
 
 interface CheckoutActions {
+  checkoutTracking: ReturnType<typeof useCheckoutTracking>
```

Comment:
On my most recent commit I moved `useCheckoutTracking` into the context for two reasons:
- It makes the tracking code much more DRY
- It means we don't have to add `mode` and `source` to every element that makes a tracking call.

There are some tests still to fix in the ExpressCheckoutUI.jest.tsx file. It's not strictly required that we use the checkout context to get the tracking helper.